pydoctor/test/test_templatewriter.py

```diff
@@ -104,7 +104,7 @@ def test_basic_package(tmp_path: Path) -> None:
         if '#' in url:
             url = url[:url.find('#')]
         assert (tmp_path / url).is_file()
-    with open(tmp_path / 'basic.html', encoding='utf-8') as f:
+    with open(tmp_path / 'index.html', encoding='utf-8') as f:
```

Comment:
Isn’t this supposed to still work with “basic.html” ?

Reply:
Yes good point. I made this change initially since I didn't create the symlinks initially. I have remove this change.